src/WhatToCook.Application/Infrastructure/Repositories/RecipesRepository.cs

```diff
@@ -94,7 +84,7 @@ public async Task<string> SaveImage(ImageInfo imageInfo)
         }
         catch (Exception exception)
         {
-            _logger.LogError($"An error occurred while saving the image. Error: {exception.Message}");
+            _logger.LogError("An error occurred while saving the image. Error:{message}", exception.Message);
```

Comment:
add whitespace between error and {message}

Reply:
will be in the next commit